### PR TITLE
Bump 23.11.0

### PR DESCRIPTION
**Pull request by jaimergp**
Checklist
* [X] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [X] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [X] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by jaimergp** [comment]
@conda-forge-admin, please rerender

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,16 +1,16 @@
...
+{% set python_version = "3.10.13" %}
 
 package:
   name: conda-standalone
   version: {{ version }}
 
 source:
   - url: https://github.com/conda/conda-standalone/archive/{{ version }}.tar.gz
-    sha256: 819fc8ee8522707a93fc15cf823bb8aa3bc57fc5255ed3efd2f61209e0ed2c86
+    sha256: 9276686c8a6ee536dc451cc6557685724fe275a44949ac4f741066fd23cdc7b4
```
Comment (jaimergp) [inline recipe/meta.yaml:13]:
```suggestion
    sha256: 2b9374d53368bffe2c010b1b659d99be7e0dcf679e916fb2b4942030b3c0712a
```

File: recipe/meta.yaml [hunk 2]
```diff
@@ -1,16 +1,16 @@
...
   name: conda-standalone
   version: {{ version }}
 
 source:
   - url: https://github.com/conda/conda-standalone/archive/{{ version }}.tar.gz
-    sha256: 819fc8ee8522707a93fc15cf823bb8aa3bc57fc5255ed3efd2f61209e0ed2c86
+    sha256: 2b9374d53368bffe2c010b1b659d99be7e0dcf679e916fb2b4942030b3c0712a
 
   - url: https://github.com/conda/conda/archive/{{ version }}.tar.gz
     sha256: 273d49db8ea723426b87866381daf2df27c7bca2c3474196460d637b62de2a1c
```
Comment (jaimergp) [inline recipe/meta.yaml:16]:
```suggestion
    sha256: 9276686c8a6ee536dc451cc6557685724fe275a44949ac4f741066fd23cdc7b4
```

**Comment by jaimergp** [comment]
Invoking [CFEP-3](https://github.com/conda-forge/cfep/blob/main/cfep-03.md) to provide a manual build of `osx-arm64`, as discussed in #54.

This is the `.ci_support/osx_arm64_.yml` file I used (not available in this repo):

```yaml
MACOSX_DEPLOYMENT_TARGET:
- '11.0'
channel_sources:
- conda-forge
channel_targets:
- conda-forge main
macos_machine:
- arm64-apple-darwin20.0.0
target_platform:
- osx-arm64
```

Then I ran this command off current `main` (with this PR freshly merged):

```
MINIFORGE_HOME=~/opt/miniforge OSX_SDK_DIR=~/opt python build-locally.py osx_arm64_ 2>&1 | tee log.txt 
```

Which produced the following output: [log.txt.zip](https://github.com/conda-forge/conda-standalone-feedstock/files/13948262/log.txt.zip)

The resulting artifact is available at https://anaconda.org/jaimergp/conda-standalone/files?channel=conda-standalone-23.11.0.

**Comment by isuruf** [comment]
Do you know why there's a `$PREFIX/bin/codesign` file?

**Comment by jaimergp** [comment]
Argh, I think it's [this symlink](https://github.com/conda-forge/conda-standalone-feedstock/blob/74bb02e11db3c26fa90c1c7398c1c846a55c9086/recipe/build.sh#L10-L15)

**Comment by jaimergp** [comment]
I should remove it in `build.sh`. Or symlink to BUILD_PREFIX instead.

**Comment by isuruf** [comment]
Yeah, it's falling back to conda-build 2 behaviour. (We need to get rid of that tech debt)

**Comment by jaimergp** [comment]
Ok, I'll add a new build. I don't think this one is broken per se, but agreed it's not ideal.

**Pull request merged**